velox/exec/HashTable.cpp

```diff
@@ -57,6 +57,7 @@ HashTable<ignoreNullKeys>::HashTable(
       pool_(pool),
       minTableSizeForParallelJoinBuild_(minTableSizeForParallelJoinBuild),
       isJoinBuild_(isJoinBuild),
+      joinBuildNoDuplicates_(!allowDuplicates),
```

Comment:
Why don't we just reuse the name allowDuplicates_ which is more straightforward? Combining with isJoinBuild_ they can tell the use case scenario

Reply:
Yes, we can reuse it by using `!allowDuplicates` to represent `noDuplicates`.